app/src/docs/v1.api-spec.yaml

```diff
@@ -401,6 +401,11 @@ paths:
         - $ref: "#/components/parameters/Query-Public"
         - $ref: "#/components/parameters/Query-MimeType"
         - $ref: "#/components/parameters/Query-Name"
+        - $ref: "#/components/parameters/Query-Permissions"
+        - $ref: "#/components/parameters/Query-Page"
+        - $ref: "#/components/parameters/Query-Limit"
+        - $ref: "#/components/parameters/Query-Sort"
+        - $ref: "#/components/parameters/Query-Order"
         - $ref: "#/components/parameters/Query-TagSet"
```

Comment:
Style: Let's preserve previous ordering by having the newly added parameters be after TagSet instead.